### PR TITLE
PLAT-306: resend outgoing after migration

### PR DESCRIPTION
**Pull request by aalekseev-ru**
**- What I did**
add resend of outgoing after migration (pulse change)

### CODE REVIEW

File: ledger-core/v2/insolar/contract/virtual.go [hunk 1]
```diff
@@ -79,3 +79,27 @@ const (
 func (f InterferenceFlag) IsZero() bool {
 	return f == 0
 }
+
+type SendResultFullFlag byte
```
Comment (sexyhamster) [inline ledger-core/v2/insolar/contract/virtual.go:83]: unused, the one that are used are in payload

Comment (aalekseev-ru) [reply ledger-core/v2/insolar/contract/virtual.go:83]: image
i named that one this way, it is unused now but will be used in future

File: ledger-core/v2/insolar/payload/vcallrequestflags.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ type SendResultFullFlag byte
 const (
 	SendResultDefault SendResultFullFlag = iota
 	SendResultFull
-	sendResultFullFlagCount = iota
+	sendResultFullFlagCount = iota - 1
```
Comment (afiskon) [inline ledger-core/v2/insolar/payload/vcallrequestflags.go:24]: The name is wrong. What you meant was probably a number starting from zero.

File: ledger-core/v2/insolar/payload/vcallrequestflags_test.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+package payload
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```
Comment (sexyhamster) [inline ledger-core/v2/insolar/payload/vcallrequestflags_test.go:14]: you do need something like 
TestCallRequestFlags_BitStorageIsEngough
that ensures, that bit storage is enough to fit your flags

Comment (aalekseev-ru) [reply ledger-core/v2/insolar/payload/vcallrequestflags_test.go:14]: https://github.com/insolar/assured-ledger/pull/229/files#diff-cf3d0de3817dffe38e53f19bf2bf656eR16-R23 why that is not enough?

Comment (sexyhamster) [reply ledger-core/v2/insolar/payload/vcallrequestflags_test.go:14]: fair enough

File: ledger-core/v2/insolar/payload/vcallrequestflags.go [hunk 4]
```diff
@@ -53,10 +51,7 @@ const (
 )
 
 func (f CallRequestFlags) WithSendResultFull(t SendResultFullFlag) CallRequestFlags {
-	if t == 0 {
-		panic(throw.IllegalValue())
-	}
-	if t > sendResultFullFlagCount {
+	if t > bitSendResultFullFlagCount {
```
Comment (afiskon) [inline ledger-core/v2/insolar/payload/vcallrequestflags.go:54]: TWIMC - this looks awful. We compare SendResultFullFlag (type '**byte**') with the **number of bits** and depending on the result doing something. I wish we had better variable names and types, but apparently this is a subject for another PR.

Comment (aalekseev-ru) [reply ledger-core/v2/insolar/payload/vcallrequestflags.go:54]: just used already approved naming https://github.com/insolar/assured-ledger/blob/master/ledger-core/v2/insolar/payload/vcallflags.go#L31

we can plan a new task about renaming and I can take it

**Pull request merged**